seichi-onp-k8s/manifests/seichi-kubernetes/app-templates/seichi-debug-minecraft-on-seichiassist-pr/templates/debug-s1/seichiassist.yaml

```
@@ -296,6 +296,11 @@ spec:
               mountPath: /plugins/Multiverse-Portals/portals.yml
               subPath: portals.yml
 
+            # WorldGuard プラグインの設定ファイル
+            - name: worldguard-config
```

```suggestion
            - name: common-mcserver-plugin-configs
```

`name`を`common-mcserver-plugin-configs`にしないとconfigを取得できないです。
あと、ClickTpaのconfigの下に移動してほしいです